api/types/container/host_config.go

```diff
@@ -216,7 +216,7 @@ type HostConfig struct {
 	SecurityOpt     []string           // List of string values to customize labels for MLS systems, such as SELinux.
 	Tmpfs           map[string]string  `json:",omitempty"` // List of tmpfs (mounts) used for the container
 	UTSMode         UTSMode            // UTS namespace to use for the container
-	ShmSize         *int64             // Total shm memory usage
+	ShmSize         int64              // Total shm memory usage
```

Comment:
Before this change, daemon will throw "SHM size must be greater then 0" error message when shmsize=0. Now, it will convert it to default 64M and keep running. Does that break API?